### ISSUE TITLE
Clarifying docs on running hybrid model with phased bam files

### ISSUE THREAD

**Issue by ASLeonard** [google/deepvariant#410]
In the hybrid model case study, the example bam files to download [here](https://github.com/google/deepvariant/blob/r1.1/docs/deepvariant-hybrid-case-study.md#download-hg003-chr20-bam) are labeled `HG003_hybrid_35x_ilmn_35x_pacb.grch38.phased.chr20.bam`. Have these bam files been phased in the same way as in the pacbio case study? If so, is the merged bam file able to be phased via `whatshap`, or is the pacbio bam phased independently and then merged with the short read bam.

Digging through the `run_deepvariant.py` script makes it seem like the phasing information only pertains to the pacbio model specifically, but just wanted to check.

Thanks,
Alex

**Comment by AndrewCarroll** [comment]
Hi @ASLeonard 

For the hybrid model, we do not use haplotagged BAMs in the training process. In our initial benchmarks of the hybrid model, we observed a similar accuracy when training with haplotag as compared to training without it. Since haplotagging is an additional step, we decided not to include haplotag in the hybrid model.

If you provide a haplotagged BAM to the hybrid model, it will still be able to read the BAM and should result in the same accuracy as when providing an untagged BAM, since the parsing of the haplotags is an additional step which is enabled by a flag.
